internal-documentation/backend-services/files-service/rest-apis.md

```diff
@@ -0,0 +1,42 @@
+# Rest APIs
+
+**Prefix**: /internal/services/files/v1/
+
+- **POST /companies/:company_id/files** Upload a file. → This route should first be called without data to initialise the entity for multi-chunk, then chunks must be sent on other route below.
+
```

Comment:
Says that it works both ways:
- Or we upload only one chunk and we can send the chunk to this route directly
- Or we plan to use multi-chunk feature and so we should use this route to initialise file (without uploading in it)

Reply:
You should also add the required / optional parameters:
?filename=...